xapian-core/queryparser/termgenerator_internal.cc

```diff
@@ -95,6 +95,47 @@ check_infix(unsigned ch)
     return 0;
 }
 
+/// check if ch is a Chinese character about digits
+static inline bool
+is_Chinese_digit(unsigned ch)
```

Comment:
Xapian convention is to not use upper case words in function names (so this should be `is_chinese_digit`).